index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="css/global.css">
+  <link rel="stylesheet" href="./css/navsection.css">
+  <link rel="stylesheet" href="./css/homesection.css">
+  <link rel="stylesheet" href="./css/programsection.css">
+  <link rel="stylesheet" href="./css/speakersection.css">
+  <link rel="stylesheet" href="./css/aboutsection.css">
+
+
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/bdd8845187.js" crossorigin="anonymous"></script>
+  <title>Masters of Philosophy</title>
+</head>
+
+<body>
+<main>
+<header>
```

Comment:
- [ ] [OPTIONAL] The header doesn't have the top dark bar, please implement this so your design look's like the provided one

|Your design|CC design|
|-|-|
|![Screenshot_20220526_203336](https://user-images.githubusercontent.com/25227022/170563788-daba09ba-ef45-4e54-8fe9-3c6574482022.png)|![Screenshot_20220526_203401](https://user-images.githubusercontent.com/25227022/170563813-65b52ac1-b0cf-49a2-b52f-663e2ec29098.png)|

Reply:
- [ ] Super cool navbar layout. Well done :+1: However, I think you missed The top short bar section with social media links, logout, and language links at the top. :grinning: 

Let's add it :point_down: 

 ![](https://camo.githubusercontent.com/96bd23868b8867313e32b0dd0bd61c02273f1ccfff54263caef020c9b2a779c2/68747470733a2f2f692e696d6775722e636f6d2f6e733846646b6b2e706e67)

Reply:
- [ ] Super cool navbar layout. Well done :+1: However, I think you missed The top short bar section with social media links, logout, and language links at the top. :grinning: 

Let's add it :point_down: 

 ![](https://camo.githubusercontent.com/96bd23868b8867313e32b0dd0bd61c02273f1ccfff54263caef020c9b2a779c2/68747470733a2f2f692e696d6775722e636f6d2f6e733846646b6b2e706e67)